### PR TITLE
[batch] make standing worker configurable in the UI

### PR DESCRIPTION
**Pull request by danking**
This particularly important for rapid iteration in dev deploys. I can enable the standing worker
while I am working. I disable it when I head off to a series of meetings and then re-enable it
when I return.

@jigold I think this will conflict with your worker pool change because I replace "1 means boolean true" with a checkbox.

### CODE REVIEW

File: batch/batch/driver/instance_pool.py [hunk 1]
```diff
@@ -85,18 +87,19 @@ async def async_init(self):
...
-  standing_worker_cores, max_instances, pool_size
+  standing_worker_cores, max_instances, pool_size, enable_standing_worker
 FROM globals;
 ''')
 
         self.worker_type = row['worker_type']
         self.worker_cores = row['worker_cores']
         self.worker_disk_size_gb = row['worker_disk_size_gb']
-        self.worker_local_ssd_data_disk = row['worker_local_ssd_data_disk']
+        self.worker_local_ssd_data_disk = bool(row['worker_local_ssd_data_disk'])
```
Comment (jigold) [inline batch/batch/driver/instance_pool.py:97]: Why do you need to add the `bool` here? Shouldn't the values coming from pymysql be booleans already?

**Review by jigold — DISMISSED** [review]

**Comment by jigold** [comment]
I added the WIP tag so you could merge at your leisure once you answer my question.

File: batch/batch/driver/instance_pool.py (hunk 1 above)
Comment (danking) [reply batch/batch/driver/instance_pool.py:97]: MySQL represents booleans as TINYINT(1), afaik. They come into Python as integers:
```python3
>>> x = l.run_until_complete(db.execute_and_fetchone('select TRUE as value'))
>>> x
{'value': 1}

```

**Review by jigold — APPROVED** [review]

**Comment by jigold** [comment]
Merge at your leisure.

**Pull request merged**